src/main/java/io/camunda/getstarted/whosfifty/CelebDetails.java

```diff
@@ -18,7 +18,7 @@ public class CelebDetails {
     @JsonProperty("height")
     public double height;
 
-    @JsonProperty("birthday")
+    @JsonProperty("birthdy") // There's no typo here :)
```

Comment:
It's really not a typo :/

API Ninjas returns `birthdy` instead of `birthday`.